.gitignore

```diff
@@ -1,2 +1,9 @@
 build/
 .ccls
+# clangd
```

Comment:
Is there a good reason for including those? I don't really mind having them. just want to know because
CMakeFiles and compile_commands.json are already included in the build folder for me.

Reply:
I don't work in the build dir hence I put the extra files into .gitignore to not commit them; what's the issue?